keyring.php

```diff
@@ -118,26 +130,26 @@ static function plugins_loaded() {
 	static function request_handlers() {
 		global $current_user;
 
-		if ( defined( 'KEYRING__FORCE_USER' ) && KEYRING__FORCE_USER && in_array( $_REQUEST['action'], array( 'request', 'verify' ) ) ) {
+		if ( defined( 'KEYRING__FORCE_USER' ) && KEYRING__FORCE_USER && in_array( $_REQUEST['action'], array( 'request', 'verify' ), true ) ) {
```

Comment:
This should be fine since $_REQUEST variables would be strings (unless someone is doing a weird filter on them I guess!)